app/Jobs/ApplicationDeploymentJob.php

```
@@ -2617,16 +2617,19 @@ private function generate_compose_file()
         if (! $this->application->custom_healthcheck_found && ! $this->application->isHealthcheckDisabled()) {
-            $docker_compose['services'][$this->container_name]['healthcheck'] = [
-                'test' => [
-                    'CMD-SHELL',
-                    $this->generate_healthcheck_commands(),
-                ],
-                'interval' => $this->application->health_check_interval.'s',
-                'timeout' => $this->application->health_check_timeout.'s',
-                'retries' => $this->application->health_check_retries,
-                'start_period' => $this->application->health_check_start_period.'s',
-            ];
+            $healthcheck_command = $this->generate_healthcheck_commands();
```

PHPMD (2.15.0)

[error] The variable $healthcheck_command is not named in camelCase. (undefined)

(CamelCaseVariableName)